tools/ios/MASTG-TOOL-0047.md

```diff
@@ -2,6 +2,9 @@
 title: Cydia
 platform: ios
 source: https://cydia.saurik.com/
+status: deprecated
+deprecation_note: Modern Jailbreaks like Dopamine and palera1n use  more modern package managers like Sileo and Zebra. Cydia has not received any meaningful update since iOS 14 and is no longer relevant on modern jailbreaks.
```

Comment:
```suggestion
deprecation_note: Modern Jailbreaks like Dopamine and palera1n use more modern package managers like Sileo and Zebra. Cydia has not received any meaningful update since iOS 14 and is no longer relevant on modern jailbreaks.
```